sausagers/src/lib.rs

```diff
@@ -1000,13 +989,13 @@ fn draw_enemy(enemy: &Enemy) {
     );
     let percent_hp = enemy.health as f32 / enemy.max_health as f32;
     let color = match percent_hp {
-        n if n <= 0.25 => 0xff0000ff,
-        n if n <= 0.5 => 0xff9900ff,
+        n if n <= 0.25 => 0xff00ff,
+        n if n <= 0.5 => 0xff00ff,
```

Comment:
Let's explicitly pass the alpha on these (add another ff at the end of both so the hex is 8 chars)